frontend/catalyst/compiler.py

```diff
@@ -77,6 +77,10 @@ class CompileOptions:
         experimental_capture (bool): If set to ``True``,
             use PennyLane's experimental program capture capabilities
             to capture the function for compilation.
+        circuit_transform_pipeline (Optional[dict[str, dict[str, str]]]):
```

Comment:
No action here, but @josh146 @isaacdevlugt this will soon beat the number of qnode's arguments.